spec/binarySearchTree.js

```diff
@@ -0,0 +1,55 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import BinarySearchTree from '../src/binarySearchTree'
+
+chai.use(chaiChange)
+
+describe('BinarySearchTree', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(BinarySearchTree).to.be.a('function')
+  })
+
+  context('insert()', () => {
+    it('inserts a node with the specified value into the tree.', () => {
+      const bst = new BinarySearchTree()
+
+      bst.insert(1)
+      bst.insert(2)
+      bst.insert(3)
+
+      expect(bst.count).to.eql(2)
```

Comment:
To `equal` - `eql` is for deep equality comparisons (e.g., comparing an array to another array). Use `equal` here.

Also: `eql` is poorly named by Chai. Use the alternative `to.deep.equal` instead.